Lib/socket.py

```diff
@@ -728,6 +728,149 @@ def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,
     else:
         raise error("getaddrinfo returns an empty list")
 
+def supports_hybrid_ipv46():
```

Comment:
Would it make sense to add a type=SOCK_STREAM parameter?

Reply:
No because `hybrid_ipv46=True` doesn't work with SOCK_DGRAM (or any other proto type AFAIK).